codalab/apps/web/templates/web/competitions/_results_page.html

```diff
@@ -81,13 +81,25 @@
                     {% for pk, scoredata in group.scores %}
                         <tr {% if phase.competition.anonymous_leaderboard and request.user.username == scoredata.username %}style="background-color: #D7FFFA;"{% endif %}>
                             <td class="row-position">
-                                {{ forloop.counter }}
+                                {% if forloop.counter < 4 %}
+                                    <b><p style="color:green;">{{ forloop.counter }}</p></b>
```

Comment:
can we get rid of the inline styling and `<b>` tag? do something like this:

`<span class="something-something">{{ forloop.counter }}</span>`

then

```
.something-something {
    font-weight: bold;
    color: green;
}```